splinepy/microstructure/tiles/cross_3d_linear.py

```
@@ -28,6 +28,23 @@ class Cross3DLinear(_TileBase):
     _n_info_per_eval_point = 1
+    _sensitivities_implemented = True
+    _closure_directions = ["z_min", "z_max"]
+    _parameters_shape = (6, 1)
+    _default_parameter_value = 0.2
+
+    # Default value of center_expansion
+    _center_expansion = 1.0
+
+    # Dynamical computation of parameter bounds depending on center expansion
+    @property
+    def _parameter_bounds(self):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_parameter_bounds`

(ANN202)